spec/aruba/command_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+RSpec.describe Aruba::Command do
+  let(:command) do
+    described_class.new(
+      'true',
+      event_bus: event_bus,
+      exit_timeout: exit_timeout,
+      io_wait_timeout: io_wait_timeout,
+      working_directory: working_directory,
+      environment: environment,
+      main_class: main_class,
+      stop_signal: stop_signal,
+      startup_wait_time: startup_wait_time
+    )
+  end
+
+  let(:event_bus) { instance_double('Aruba::EventBus') }
+  let(:exit_timeout) { 1 }
+  let(:io_wait_timeout) { 1 }
+  let(:working_directory) { expand_path('.') }
+  let(:environment) { ENV.to_hash }
+  let(:main_class) { nil }
+  let(:stop_signal) { nil }
+  let(:startup_wait_time) { 1 }
+
+  context '#started' do
+    before :each do
+      allow(event_bus).to receive(:notify).with(Events::CommandStarted)
+    end
```

Comment:
`before :each` => `before` (this is the default; and it has been renamed in newer versions of RSpec; [`:each` is now `:example`](https://relishapp.com/rspec/rspec-core/v/3-7/docs/hooks/before-and-after-hooks))